docs/_includes/footer.html

```diff
@@ -17,8 +17,14 @@
 
     <div class="large-4 columns">
       <ul class="social icons">
-        <li><a href="//twitter.com/capistranorb"><i class="foundicon-twitter"></i></a></li>
-        <li><a href="//github.com/capistrano"><i class="foundicon-github"></i></a></li>
+        <li class="foundicon"><a href="//twitter.com/capistranorb"><i class="foundicon-twitter"></i></a></li>
+        <li class="foundicon"><a href="//github.com/capistrano"><i class="foundicon-github"></i></a></li>
+        <li class="thanks-dnsimple">
+          <a href="http://dnsimple.link/resolving-hanami" target="_blank">
```

Comment:
damn, copy-paste mistake :) they didn't provide me with a link to use. I'll direct Simone at DNSimple to this issue and he can perhaps provide us one they'd like us to use for their own traffic metrics.